python/versions.bzl

```diff
@@ -45,7 +36,11 @@ TOOL_VERSIONS = {
         },
     },
     "3.8.12": {
-        "url": "20220227/cpython-{python_version}+20220227-{platform}-{build}.tar.gz",
+        "url": {
+            "x86_64-apple-darwin": "20220227/cpython-{python_version}+20220227-{platform}-{build}.tar.gz",
```

Comment:
these still look the same, is there a reason to make url a dict now instead of a string?

Reply:
From what @indygreg mentioned in https://github.com/indygreg/python-build-standalone/issues/118#issuecomment-1066180403, I was expecting the `aarch64-apple-darwin` platform to have a different release to the others, so this entry will eventually need to be a dict.